thirdparty/build-thirdparty.sh

```diff
@@ -1319,6 +1319,17 @@ build_simdutf() {
     ${BUILD_SYSTEM} install
 }
 
+# tenann
+build_tenann() {
+    check_if_source_exist $TENANN_SOURCE
+    rm -rf $TP_INSTALL_DIR/include/tenann
+    rm -rf $TP_INSTALL_DIR/lib/libtenann-bundle.a
+    rm -rf $TP_INSTALL_DIR/lib/libtenann-bundle-avx2.a
+    cp -r $TP_SOURCE_DIR/$TENANN_SOURCE/include/tenann $TP_INSTALL_DIR/include/tenann
+    cp -r $TP_SOURCE_DIR/$TENANN_SOURCE/lib/libtenann-bundle.a $TP_INSTALL_DIR/lib/
+    cp -r $TP_SOURCE_DIR/$TENANN_SOURCE/lib/libtenann-bundle-avx2.a $TP_INSTALL_DIR/lib/
```

Comment:
this prebuilt libraries won't work for multiple platforms, remember that we now support both x86 and arm platform.

Reply:
you are right, we will build an additional libraries for the ARM platform.

Reply:
Does the x86 platform contain Mac Intel? I test to build the tenann source code in mac intel, It occurs some error

Reply:
it's known to be broken on Mac, starrocks can be built natively on Mac for now.